pyIGTLink/pyIGTLink.py

```diff
@@ -180,26 +217,47 @@ def __init__(self):
         self._timestamp = time.time() * 1000
 
         self._endian = ">"  # big-endian
+        self.crc = None
 
         self._binary_body = None
         self._binary_head = None
         self._body_pack_size = 0
+        self._body_size = 0
 
     def pack(self):
         binary_body = self.get_binary_body()
-        body_size = self.get_body_pack_size()
-        crc = CRC64(binary_body)
+        self._body_size = self.get_body_pack_size()
+        self.crc = CRC64(binary_body)
```

Comment:
Now you use both self._crc and self.crc, so we should at least only use self._crc.

But if we currently do not use self._crc and self._head, I do not see any reason for keeping it. We could still return crc in dict in line 262 of this file.